src/protocols/secure_channel/PASESession.cpp

```diff
@@ -979,4 +981,31 @@ CHIP_ERROR PASESession::OnMessageReceived(ExchangeContext * exchange, const Payl
     return err;
 }
 
+CHIP_ERROR PASEVerifier::Serialize(PASEVerifierSerialized & output)
```

Comment:
Can we adopt the approaches from https://github.com/project-chip/connectedhomeip/pull/14667 to make writes to these output buffers safer?

Reply:
done